release_notes.md

```diff
@@ -7,7 +7,7 @@
 - Update Python Worker Version to [1.1.12](https://github.com/Azure/azure-functions-python-worker/releases/tag/1.1.12)
 - Updated Node.js Worker Version to [2.1.0](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v2.1.0)
 - Updated Java Worker Version to [1.8.1](https://github.com/Azure/azure-functions-java-worker/releases/tag/1.8.1)
-- Updated PowerShell Worker to [3.0.629](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.629) (PS7) and [3.0.630](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.630) (PS6)
+- Updated PowerShell Worker to [3.0.705](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.705) (PS7) and [3.0.630](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.630) (PS6)
```

Comment:
I think we actually need to append a new line to the end of this file, not edit the old line. See the Java worker update, for example

Reply:
> I think we actually need to append a new line to the end of this file, not edit the old line. See the Java worker update, for example

Good catch, thanks @AnatoliB! 

Fixed. Could you please take another look?